SpatialGDK/Source/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -91,6 +92,10 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	USpatialActorChannel* GetActorChannelByEntityId(Worker_EntityId EntityId) const;
 
+	DECLARE_DELEGATE(ServerTravelDelegate);
+
+	void WipeWorld(const USpatialNetDriver::ServerTravelDelegate& LoadSnapshotAfterWorldWipe);
```

Comment:
Should this delegate variable be called something like PostWorldWipeDelegate? It's up to the user what they want to do.

Reply:
Sure you're not wrong there.